src/out/OutSimPack2.ts

```diff
@@ -0,0 +1,101 @@
+import {
+  array,
+  byte,
+  float,
+  int,
+  string,
+  struct,
+  unsigned,
+} from '../decorators';
+import { Struct } from '../packets';
+import { OutSimInputs } from './OutSimInputs';
+import { OutSimMain } from './OutSimMain';
+import { OutSimOptions } from './OutSimOptions';
+import { OutSimWheel } from './OutSimWheel';
+
+export class OutSimPack2 extends Struct {
+  // if (OSOpts & OSO_HEADER)
+  @string(4) Header = '';
+  // if (OSOpts & OSO_ID)
+  @int() ID = 0;
+  // if (OSOpts & OSO_TIME)
+  @unsigned() Time = 0;
+  // if (OSOpts & OSO_MAIN)
+  @struct(OutSimMain) OSMain = new OutSimMain();
+  // if (OSOpts & OSO_INPUTS)
+  @struct(OutSimInputs) OSInputs = new OutSimInputs();
+  // if (OSOpts & OSO_DRIVE)
+  @byte() Gear = 0; // 0=R, 1=N, 2=first gear
```

Comment:
Please use the JSDoc style comments for annotating the properties, like so:

```ts
/** 0=R, 1=N, 2=first gear */
@byte() Gear = 0;


```

then they will be shown correctly in IDE documentation:

<img width="339" alt="image" src="https://user-images.githubusercontent.com/6420535/229872728-fc188758-c199-4034-a9b6-202862fb0f90.png">

Reply:
converted documentation to proposed format